db/migrate/20150813185339_create_subscriptions.rb

```diff
@@ -0,0 +1,73 @@
+class CreateSubscriptions < ActiveRecord::Migration
+  def change
+    create_table "katello_subscriptions" do |t|
+      t.string :product_name
+      t.string :product_id
```

Comment:
These are candlepin fields, which are related to the actual subscription, not a product. Product name is actually the subscription name, and product ID is another candlepin id like "SER0409" for a subscription. I could change "product name" to "name" and "product id" to something else? One way is confusing because we have a product model, and changing them doesn't reflect what is in the candlepin db. thoughts?